droidlet/tools/hitl/nsp_retrain_infra.py

```diff
@@ -161,28 +161,39 @@ def create_mask(self, opts, batch_id, data_filepath):
         # Create train, valid, and test masks based on user input
         total_rows = sum(1 for line in open(data_filepath))
         logging.info(f"Model training data masks are being generated:")
-        if  not opts.resample:
-            logging.info(f"Model will be trained with old+new training data, valid/test are static old data (not resampled)")
-
-            # Lengthen valid and test masks to be the length of all data
-            old_mask_filepath = os.path.join(opts.sweep_config_folder, 'split_masks.pth')
-            old_masks = torch.load(old_mask_filepath)['annotated']
-            valid_mask = [x for x in old_masks['valid']]
-            valid_mask.extend([False] * (total_rows - len(valid_mask)))
-            test_mask = [x for x in old_masks['test']]
-            test_mask.extend([False] * (total_rows - len(test_mask)))
-            train_mask = [False if valid_mask[i] or test_mask[i] else True for i in range(total_rows)]  # Rest of the data are train
-        else:
-            logging.info(f"Model train/valid/test data will be resampled from combined old+new dataset in 80/10/10 split")
-
-            # Generate random splits and populate masks
-            mask_designation = ['test'] * int(total_rows * 0.1)
-            mask_designation.extend(['valid'] * int(total_rows * 0.1))
-            mask_designation.extend(['train'] * (total_rows - len(mask_designation)))
-            random.shuffle(mask_designation)
-            train_mask = [True if x == 'train' else False for x in mask_designation]
-            valid_mask = [True if x == 'valid' else False for x in mask_designation]
-            test_mask = [True if x == 'test' else False for x in mask_designation]
+
+        # Old masks are static, just extend to be the length of the new dataset
+        old_mask_filepath = os.path.join(opts.sweep_config_folder, 'split_masks.pth')
+        old_masks = torch.load(old_mask_filepath)['annotated']
+        old_train_mask = [x for x in old_masks['train']]
```

Comment:
isn't this the same as : `old_train_mask = old_masks['train']` why iterate over and build a list ? 
Same for valid and test

Reply:
To convert from a torch tensor to a list, but there's also a built in function to do it.